services/spar/src/Spar/Sem/SamlProtocolSettings.hs

```diff
@@ -21,17 +21,20 @@ module Spar.Sem.SamlProtocolSettings
   ( SamlProtocolSettings (..),
     spIssuer,
     responseURI,
+    contactPersons,
   )
 where
 
+import Data.Domain
 import Data.Id (TeamId)
 import Imports
 import Polysemy
 import qualified SAML2.WebSSO.Types as SAML
 import qualified URI.ByteString as URI
 
 data SamlProtocolSettings m a where
-  SpIssuer :: Maybe TeamId -> SamlProtocolSettings m SAML.Issuer
-  ResponseURI :: Maybe TeamId -> SamlProtocolSettings m URI.URI
+  SpIssuer :: Maybe TeamId -> Maybe Domain -> SamlProtocolSettings m (Maybe SAML.Issuer)
+  ResponseURI :: Maybe TeamId -> Maybe Domain -> SamlProtocolSettings m (Maybe URI.URI)
+  ContactPersons :: Maybe Domain -> SamlProtocolSettings m ([SAML.ContactPerson])
```

Comment:
why no team id for contact persons?

Reply:
m|  answer: contact person does not return a URI containing the team id.  thanks!  :)